tasks/2-build-tag-push-mq.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
I see a lot of code duplication, we have other tasks that build tag and push, what's the difference?